plugins/scroll-options/src/utils.js

```diff
@@ -11,14 +11,10 @@
  * @returns {!Blockly.utils.Coordinate} The current workspace coordinate.
  */
 export const getTranslation = (ws) => {
-  const dragSurface = ws.getBlockDragSurface();
-  if (dragSurface) {
-    return dragSurface.getWsTranslation();
-  } else {
-    const translation = ws.svgBlockCanvas_.getAttribute('transform');
-    const splitted = translation.split(',');
-    const x = Number(splitted[0].split('(')[1]);
-    const y = Number(splitted[1].split(')')[0]);
-    return new Blockly.utils.Coordinate(x, y);
-  }
+  // TODO: We should maybe make getBlockCanvas public?
+  const translation = ws.svgBlockCanvas_.getAttribute('transform');
+  const splitted = translation.split(',');
+  const x = Number(splitted[0].split('(')[1]);
+  const y = Number(splitted[1].split(')')[0]);
```

Comment:
I think this deserves a comment with an example of what format `translation` is in.

(Also, consider if this is a place where a regexp with capture groups might actually be easier to read.)

Reply:
Done!